packages/material-ui/src/StyledEngineProvider/index.js

```diff
@@ -1 +0,0 @@
-export { StyledEngineProvider as default } from '@material-ui/styled-engine';
```

Comment:
This is going to break many of the codesandbox we have created with v5 and that are not on a fixed version of next. How about keeping it but with a deprecation message? (so we can remove it during the lifecycle of v5.x

Reply:
Ah, I left it in `packages/material-ui/src`, but removed it here. Yeah it makes sense, let me do that